src/uct/base/uct_iface.c

```diff
@@ -150,6 +150,36 @@ static ucs_status_t uct_base_iface_flush(uct_iface_h tl_iface)
     return UCS_OK;
 }
 
+static ucs_status_t uct_iface_nop_iface_arg(uct_iface_h tl_iface)
+{
+    return tl_iface->status;
+}
+
+static ucs_status_t uct_iface_nop_ep_arg(uct_ep_h ep)
+{
+    return ep->iface->status;
+}
+
+void uct_base_iface_set_error(uct_iface_h tl_iface, ucs_status_t status)
+{
+    void **func_iter;
+    tl_iface->status = status;
+
+    /* Overwrite functions with endpoint as first argument */
```

Comment:
minor: seems to be a typo in the comment, because functions with iface argument are from the beginning.